ddr/include/ddr/scanner/dwarf/DwarfFunctions.hpp

```diff
@@ -42,6 +42,8 @@
 #include <sys/errno.h>
 #endif /* AIXPPC */
 
+const char * scanFileName =NULL ;
```

Comment:
I think a better place for this is a static member of `DwarfScanner`.
Also please fix the formatting.

Reply:
well, you know, looks like the file  AixSymbolTableParser.cpp and DwarfFunctions.hpp are used to implement some dwarf functions, which are available in libdwarf.so on linux. And currently, we didn't do any reference to DwarfScanner class in AixSymbolTableParser.cpp. Do you think we want to import that class into this object as well?

Reply:
`DwarfFunctions.hpp` should declare only features that are normally available via in `libdwarf.so` on other platforms. Here, we're discussing an additional communication channel between `DwarfScanner` and `AixSymbolTableParser`, so, yes, I believe `DwarfScanner` is the right place for that. An improvement to my original proposal would be to make that additional field of `DwarfScanner` private and provide an accessor function for use in `AixSymbolTableParser`.

Reply:
understand .  Do you want to keep that new field static ? If static , I would like to make it in public without access functions. If private and object level of field, then looks like we need to find a way to store the "this" object , as we don't want to change params of dwarf_init() , do we? 
So, looks like a public static is more reasonable. agree?

Reply:
I think private & static with an accessor is better so it's clear that only `DwarfScanner` has write access.
```
private:
    static const char * scanFileName;
public:
    static const char * getScanFileName() { return scanFileName; }
```

Reply:
done. Please review. thx.